wrangler-core/src/main/java/io/cdap/functions/GeoFences.java

```diff
@@ -47,7 +47,7 @@
    * @param geofences GeoJson representation of the fence area
    * @return true if location is inside any of the given geofences, else false
    */
-  public static Boolean inFence(double latitude, double longitude, String geofences) {
+  public static Boolean InFence(double latitude, double longitude, String geofences) {
```

Comment:
this is a backwards incompatible change, unless this method didn't exist before

Reply:
Added back old methods and deprecated it.